src/internet_identity/src/storage.rs

```diff
@@ -716,20 +716,17 @@ impl<M: Memory + Clone> Storage<M> {
             Some(Principal::self_authenticating(&device.pubkey))
         };
 
-        let previous_devices = previous_devices
+        let previous_recovery_phrases = previous_devices
```

Comment:
Do the changes in this function change the functionality of the function or are they only to make it more readable?